vuepress/docs/next/docs/compose/local-hub-overview.md

```diff
@@ -17,9 +17,9 @@ code that can be used in an Entando widget, page or
 application. Examples of components are widgets, micro frontends,
 content types, labels, plugins, and static resources.
 
-### ECR Bundle
+### Local Hub Bundle
 
-An ECR bundle is a package containing one or more components and a
+A Local Hub bundle is a package containing one or more components and a
 descriptor.yaml file providing information about the bundle. The
 bundle is published in a Git registry and shared with an Entando
```

Comment:
We need to add a TODO to check for language like this. We're switching to v5 which means bundles are published to docker.

Reply:
@nshaw modified here to briefly touch on both git- and docker-based